README.md

```diff
@@ -23,26 +23,32 @@ The compiled extension is installed with: `make install` (which requires superus
 The extension is also available on [PGXN](https://pgxn.org/dist/pg_diffix/), and can be installed using
 [PGXN Client](https://pgxn.github.io/pgxnclient/).
 
-When connected to the target database with `psql`, you have to set up the extension with `CREATE EXTENSION pg_diffix;`.
+## Activating the extension
 
-## Using the extension
+You can set up the extension for the current database by using the command `CREATE EXTENSION pg_diffix;`.
 
-Load the extension with `LOAD 'pg_diffix';`, unless you configured it to preload using [these instructions](#preloading-the-extension).
+To properly enforce the anonymization restrictions, the extension has to be automatically loaded on
+every session start for restricted users. This can be accomplished by configuring
+[library preloading](https://www.postgresql.org/docs/current/runtime-config-client.html#RUNTIME-CONFIG-CLIENT-PRELOAD).
+
+For example, to automatically load the `pg_diffix` extension for all users connecting to a database,
+you can execute the following command:
+
+`ALTER DATABASE db_name SET session_preload_libraries TO 'pg_diffix';`
```

Comment:
Is this setting persistent? If so, where is it stored?

Reply:
Yes, it is persistent. Not sure where it is stored, I assume in the database metadata or similar.